nextcord/http.py

```diff
@@ -330,7 +333,16 @@ async def request(
                         data = await json_or_text(response)
 
                         # check if we have rate limit header information
-                        remaining = response.headers.get("X-Ratelimit-Remaining")
+                        is_global = bool(response.headers.get("X-RateLimit-Global", False))
```

Comment:
Just plays nicer with types (at-least in pycharm)

```suggestion
                        is_global = bool(response.headers.get("X-RateLimit-Global", ""))
```

Reply:
Can you elaborate how this plays nicer with types? I don't see how the current code would cause a type error.